packages/utils/src/lib/git.spec.ts

```diff
@@ -0,0 +1,9 @@
+import { latestHash } from './git';
+import { expect } from 'vitest';
+
+describe('git', () => {
+  it('should log current hash', async () => {
+    const hash = await latestHash();
+    expect(hash.length).toBeGreaterThan(0);
```

Comment:
We can assert more than just if the hash is non-empty. A git hash is always a 40-character hexadecimal string, which we can easily check.

```suggestion
    expect(hash).toHaveLength(40);
    expect(hash).toMatch(/^[0-9a-f]+$/);
```